pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
+	pvcs := &core.PersistentVolumeClaimList{}
+	pvcLabels := map[string]string{
+		"migration": r.Migration.Name,
```

Comment:
don't we have also vmID label? that would help to minimize the PVC list and select the specific PVC of a VM we are now looking into. here we will get a list of all the PVCs associated with the migration.

Reply:
we didn't but I've added it now since it make more sense.